iree/compiler/Pipelines/Options.cpp

```diff
@@ -38,8 +38,10 @@ void InputDialectOptions::bindOptions(OptionsBinder &binder) {
                      "Legalize from TOSA ops."),
           clEnumValN(InputDialectOptions::Type::mhlo, "mhlo",
                      "Legalize from MHLO ops."),
+#ifdef IREE_HAVE_TORCH_MLIR_DIALECTS
```

Comment:
It seems this is not turned on. I am not sure why. The build target does depend on https://github.com/google/iree/blob/37f0e7dc332de6a4ff36ea1ec08e3fc0b4af77b0/iree/compiler/Pipelines/CMakeLists.txt#L51 which defines this preprocessor variable.

Reply:
The original failure is unrelated to this though.